program-workflows/how-to-enter-recency-data.md

```diff
@@ -0,0 +1,23 @@
+##HOW TO ENTER RECENCY DATA WITHIN THE HTS CLIENT CARD
+
+###Accessing the HTS client card for an existing or new patient in the EMR
+
+1.	Search for the given patient or create a new patient
+
+2.	Open a recent visit or start a past visit for the newly created patient
+
+3.	View the patient’s dashboard and look for the HMIS ACP 018:HTS CLIENT CARD that shall be used to enter in the data.
+
+![HTS Client Card](../images/hts-client-card018.jpg)
+
+###Things to note while entering the client card.
```

Comment:
@ODORA0 Just call this Notes 

Also add a space between the hashes and the text (global)

Reply:
@ssmusoke Done

Reply:
@ODORA0 Change from a heading to just a Bold title